.github/workflows/rust.yml

```diff
@@ -26,30 +26,37 @@ jobs:
     - name: Build
       working-directory: ${{ matrix.workdir }}
       run: cargo build --verbose
+      if: always()
```

Comment:
Ensure if someone forgot to format, and docs, and has live lints, that they use $O(n)$ rather than $O(n^2)$ CI time.